src/components/Sidebar/SidebarHeader.tsx

```diff
@@ -85,13 +85,16 @@ const SidebarHeader: FC<SidebarHeaderProps> = ({
                 onKeyPress={handleKeyPress}
                 defaultValue={currentNodeLabel}
                 autoFocus
+                readOnly={currentNodeKey === 'Start'}
```

Comment:
This makes the input read only, but we really don't want the input to show up at all if it can't be edited. To fix, remove this line, and add `&& currentNodeKey !== 'Start'` to the check on line 78.

We also want to get rid of the pointer cursor since it is no longer interactive. To do so, change the className on the header label div on line 93 to:

```js
className={clsx(
  styles.headerLabel,
  styles.headerLabelText,
  currentNodeKey === 'Start' && styles.headerLabelStart
)}
```

Then add the following style to the sidebar stylesheet: 

```js
headerLabelStart: {
  cursor: 'default'
}
```

Reply:
Prevent start from expanding and made these changes.